### PR TITLE
BO-4100:  DummyMarketplacePayment module integration guide

### PR DESCRIPTION
**Pull request by az-joss**
## PR Description

Added DummyMarketplacePayment module integration guide

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by ilyakubanov — COMMENTED** [review]
- The link will not work, please merge master and move a new guide to 202108.0 folder
- Add related config from https://github.com/spryker/suite-nonsplit/pull/3809/files

File: _data/sidebars/marketplace_dev_sidebar.yml [hunk 1]
```diff
@@ -53,6 +53,9 @@ entries:
 
       - title: Feature integration guides
         nested:
+          - title: Marketplace Dummy Payment feature integration
+            url: /docs/marketplace/dev/feature-integration-guides/marketplace-dummy-payment-feature-integration.html
+
```
[the remark this answers is not in the document]
Comment (az-joss) [reply _data/sidebars/marketplace_dev_sidebar.yml:58]: moved

File: docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+Follow the steps below to install the Marketplace Dummy Payment feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | INTEGRATION GUIDE |
+| - | - | - |
+| Spryker Core | 202001.0   | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| Marketplace Merchant | dev-master | [Marketplace Merchant feature integration](/docs/marketplace/dev/feature-integration-guides/{{ page.version }}/marketplace-merchant-feature-integration.html)
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md:21]: added

File: docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+{% info_block warningBox "Verification" %}
+
+Make sure that the following transfer objects are generated:
+
+| TRANSFER | TYPE | EVENT  | PATH  |
+| --------- | ------- | ----- | ------------- |
+| DummyMarketplacePayment | class | created | src/Generated/Shared/Transfer/DummyMarketplacePayment |
+| Payment.dummyMarketplacePaymentInvoice | attribute | created | src/Generated/Shared/Transfer/Payment |
+| Order.dummyMarketplacePaymentInvoice | attribute | created | src/Generated/Shared/Transfer/Order |
+| Quote.payment | attribute | created | src/Generated/Shared/Transfer/Quote |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md:58]: removed

File: docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following transfer objects are generated:
+
+| TRANSFER | TYPE | EVENT  | PATH  |
+| --------- | ------- | ----- | ------------- |
+| DummyMarketplacePayment | class | created | src/Generated/Shared/Transfer/DummyMarketplacePayment |
+| Payment.dummyMarketplacePaymentInvoice | attribute | created | src/Generated/Shared/Transfer/Payment |
+| Order.dummyMarketplacePaymentInvoice | attribute | created | src/Generated/Shared/Transfer/Order |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md:57]: renamed

File: docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+### 3) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN | DESCRIPTION | PREREQUISITES | NAMESPACE |
+| --------- | ------- | ----- | ------------- |
+| MerchantProductItemPaymentMethodFilterPlugin | Filters PaymentMethodTransfer with DummyMarketplacePaymentProvider from the PaymentMethodsTransfer if QuoteTransfer has not only merchant items. |  | Spryker\Zed\DummyMarketplacePayment\Communication\Plugin\Payment\MerchantProductItemPaymentMethodFilterPlugin |
+| DummyMarketplacePaymentHandlerPlugin | Adds step that expands QuoteTransfer with DummyMarketplacePayment provider name and method name to StepHandlerPluginCollection. |  | Spryker\Yves\DummyMarketplacePayment\Plugin\StepEngine\DummyMarketplacePaymentHandlerPlugin |
+| DummyMarketplacePaymentInvoiceSubFormPlugin | Adds InvoiceSubForm and its data provider to SubFormPluginCollection. |  | Spryker\Yves\DummyMarketplacePayment\Plugin\StepEngine\SubForm\DummyMarketplacePaymentInvoiceSubFormPlugin |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202106.0/marketplace-dummy-payment-feature-integration.md:70]: replaced

**Comment by az-joss** [comment]
> * The link will not work, please merge master and move a new guide to 202108.0 folder
> * Add related config from https://github.com/spryker/suite-nonsplit/pull/3809/files

- master is merged
- added configs, import, and YVES translation sections

**Review by ilyakubanov — APPROVED** [review]

**Pull request merged**